swimmer.py

```diff
@@ -213,4 +213,48 @@ def get_reward(self, physics):
     return rewards.tolerance(physics.nose_to_target_dist(),
                              bounds=(0, target_size),
                              margin=5*target_size,
-                             sigmoid='long_tail')
\ No newline at end of file
+                             sigmoid='long_tail')
+
+class Swimmer(control.Task):
+    def __init__(self, arena_size=(1, 1)):
+        self.arena_size = arena_size
+        self.food_zone_size = min(arena_size) / 10
+        self.food_zone_pos = self._random_food_zone()
+        self.sensor_model = FFNSensor()
```

Comment:
If you're referring to the head-sensor we were talking about - you could remove it from here, we rather have it in the Actor, along with the other parts of the model.